pkg/validation/internal/good_practices_test.go

```diff
@@ -0,0 +1,67 @@
+package internal
+
+import (
+	"github.com/operator-framework/api/pkg/manifests"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func Test_ValidateGoodPractices(t *testing.T) {
+	type args struct {
+		bundleDir      string
+	}
+	tests := []struct {
+		name        string
+		args        args
+		wantError   bool
+		wantWarning bool
+		errStrings  []string
+		warnStrings []string
+	}{
+		{
```

Comment:
Is it th exercising the other error paths as well, e.g. when the bundle is nil, or the bundle.CSV is nil, etc.

Reply:
also when `csv.Spec.InstallStrategy.StrategySpec.DeploymentSpecs` is nil

Reply:
Ideally yes, I agree.
Good point done. 🥇 
thank you